### PR TITLE
faq styles

### PR DESCRIPTION
**Pull request by cgbl-90**

### CODE REVIEW

File: src/content/storybook/setup.mdx [hunk 1]
```diff
@@ -362,3 +362,20 @@ Use the same method to debug the following errors:
...
+
+If styles are not rendering correctly, there could be several reasons behind that:
+
+1. **The order in which stylesheets are loaded.** In development mode, styles from the preview might be imported earlier than component styles, while in production mode, this order could be reversed. This affects the specificity and, ultimately, the rendering of styles. If this is the case, the styles render but are overwritten by others. 
+
+Note that stylesheets were **initially** loaded in Storybook 6 and older, while from Storybook 7 onwards, they only get loaded **when the story is viewed**. This affects how styles load.
+
+2. **Browser compatibility.** If the browser version does not support certain CSS features, styles may not apply correctly. Find browser versions supported by our Capture Infrastructure [here](/docs/infrastructure-release-notes).
+
+3. **Shadow DOM.** If you use web components with styles encapsulated in a shadow DOM, Chromatic's capture helpers may not capture them.
```
Comment (winkerVSbecks) [inline src/content/storybook/setup.mdx:377]: @cgbl-90 not sure what this means. Could you share an example?

Comment (winkerVSbecks) [reply src/content/storybook/setup.mdx:377]: My understanding was that shadow DOM CSS should work fine

Comment (winkerVSbecks) [reply src/content/storybook/setup.mdx:377]: Yea, those are capturing issues. Styles still render fine.

Comment (cgbl-90) [reply src/content/storybook/setup.mdx:377]: The styles won't be reflected in the snapshot @winkerVSbecks Can you rewrite it to include it in this explanation?

**Pull request merged**